samples/basic/fade_led/src/main.c

```diff
@@ -17,12 +17,12 @@
 #define PWM_LED0_NODE	DT_ALIAS(pwm_led0)
 
 #if DT_NODE_HAS_STATUS(PWM_LED0_NODE, okay)
-#define PWM_LABEL	DT_PWMS_LABEL(PWM_LED0_NODE)
+#define PWM_CTLR	DT_PWMS_CTLR(PWM_LED0_NODE)
 #define PWM_CHANNEL	DT_PWMS_CHANNEL(PWM_LED0_NODE)
 #define PWM_FLAGS	DT_PWMS_FLAGS(PWM_LED0_NODE)
 #else
 #error "Unsupported board: pwm-led0 devicetree alias is not defined"
-#define PWM_LABEL	""
+#define PWM_CTLR
```

Comment:
Wouldn't this result in a syntax error in `DEVICE_DT_GET()`?  IMO defining it to `DT_INVALID_NODE` would be a better idiom.

Reply:
Forgot about `DT_INVALID_NODE`.

Reply:
> Wouldn't this result in a syntax error in `DEVICE_DT_GET()`?

I thought so too, but I checked before approving and it did not.

That makes sense in retrospect: it's not a syntax error because it expands to a declaration for a device that does not exist. That would ordinarily result in a linker error eventually, but we never get there because the compilation is halted by the `#error`.

>  Forgot about DT_INVALID_NODE.

That said, I don't object to this.